src/components/Ingredients/Ingredients.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { fireEvent, render } from '@testing-library/react';
+import Ingredients from './Ingredients';
+
+describe('<Ingredients/>', () => {
+  const recipeId = 1;
+  const ingredients = [{
+    id: 0,
+    name: 'tomato',
+  }, {
+    id: 1,
+    name: 'salt'
+  }];
+
+  it('should render the list of ingredients component', () => {
+    const editIngredient = jest.fn();
+    const removeIngredient = jest.fn();
+
+    const { getByText } = render(
+      <Ingredients
+        recipeId={recipeId}
+        ingredients={ingredients}
+        editIngredient={editIngredient}
+        removeIngredient={removeIngredient}
+      />);
+
+    ingredients.forEach(ingredient => {
+      expect(getByText(ingredient.name)).toBeVisible();
+    });
+  });
+
+  it('should remove the ingredient when deleted from the EditDeleteBar', () => {
+    const editIngredient = jest.fn();
+    const removeIngredient = jest.fn();
+
+    const { getByRole, getByText } = render(
+      <Ingredients
+        recipeId={recipeId}
+        ingredients={ingredients}
+        editIngredient={editIngredient}
+        removeIngredient={removeIngredient}
+      />);
+
+
+    const ingredient = getByText(ingredients[0].name);
+    fireEvent.mouseOver(ingredient);
+    const remove = getByRole('button', { name: /edit/i });
+    fireEvent.click(remove);
+
+    expect(ingredient).not.toBeVisible();
```

Comment:
As a rule of thumb, I think it's preferred to use:
* `expect(element).toBeVisible()` when something is supposed to be shown to the user.
* `expect(element).not.toBeInTheDocument()` when something it's not supposed to be there (like in this case).

Details [here](https://stackoverflow.com/a/69854374/4838227) and in the [docs](https://github.com/testing-library/jest-dom#tobevisible).

Reply:
Noted!